crates/fuel-gas-price-algorithm/src/v1.rs

```diff
@@ -60,17 +60,17 @@ pub struct AlgorithmV1 {
     /// The latest known cost per byte for recording blocks on the DA chain
     latest_da_cost_per_byte: u128,
     /// The cumulative reward from the DA portion of the gas price
-    total_rewards: u64,
+    total_rewards: u128,
     /// The cumulative cost of recording L2 blocks on the DA chain as of the last recorded block
     total_costs: u128,
     /// The P component of the PID control for the DA gas price
     da_p_factor: i64,
     /// The D component of the PID control for the DA gas price
     da_d_factor: i64,
```

Comment:
is my understanding correct that these values are `i64`'s because we need them to be for float calculation?

Reply:
I'm not sure what you mean by "float calculation". We don't have any floats in this code.

These actually probably don't need to be `i` and could be `u`. When I added them originally, I was open to the idea of using negative values. Just haven't reevaluated.

Reply:
ah okay. as far as i remember there were a bunch of floats used so i might be wrong